lib/beaker/hypervisor/openstack.rb

```diff
@@ -152,20 +152,13 @@ def cleanup
       end
     end
 
-    # Enables root for instances with custom username like ubuntu-amis
-    #
-    # @return [void]
-    # @api private
-    def enable_root_on_hosts
-      @hosts.each do |host|
-        enable_root(host)
-      end
-    end
-
     # Enables root access for a host when username is not root
     #
     # @return [void]
     # @api private
+    def enable_root_on_hosts(host)
```

Comment:
So, this still has a method name that allows for 'hosts' but you've changed the signature to accept a single host.  It then wraps a method that does take a single host.  I'm pretty confused as to the purpose of this.